t/e2e/transactions/transactions_test.go

```diff
@@ -436,6 +453,9 @@ func (s *TransactionsTestSuite) TestSendEtherTxUpstream() {
 }
 
 func (s *TransactionsTestSuite) TestDoubleCompleteQueuedTransactions() {
+	if SkipTransactionTest(BothNetworks) {
```

Comment:
`MainnetOnly`, there was no exception for StatusChain before. And the rest test cases in this file.